src/nidigital/metadata/functions.py

```diff
@@ -2085,6 +2086,7 @@
         'returns': 'ViStatus'
     },
     'LoadSpecifications': {
+        'codegen_method': 'private',
         'documentation': {
             'description': 'TBD'
         },
```

Comment:
I assume we can remove documentation from private methods?

Reply:
I think so. Created #1405 to address it.